app/services/disability_claim_service.rb

```diff
@@ -33,6 +33,15 @@ def request_decision(id)
     client.submit_5103_waiver(id).body
   end
 
+  # upload file to s3 and enqueue job to upload to EVSS
+  # TODO (AJM): Unique filename in s3 but preserve original filename when uploading to evss
+  # TODO (AJM): Give user GlobalID for serialization
```

Comment:
So one thought I had for something like this is that any CRUD operation that does not actually result in a an actual "success" would instead return a '203 Accepted' and the json payload would have a job id associated with it. Then we could have an 'unsecured' jobs endpoint (unsecured in terms of id.me integration) that would inform whether or not a job was successfully completed that could polled for any notifications, or later tied in to notifications somehow. This endpoint would never return any user identifying information -- just job specific notification of succeeded at such and such time.